base_classes/NXsample.nxdl.xml

```diff
@@ -328,6 +326,116 @@
 	<field name="thickness" type="NX_FLOAT" units="NX_LENGTH">
 		<doc>sample thickness</doc>
 	</field>
+	<field name="sample_id" type="NX_CHAR">
```

Comment:
Where should such a sample_environment be placed, and what would it contain? I don't really see the big problem with the list of items in NXsample, other that it might eventually become hard to find the item you are looking for in the base class.
If we make a separate NXsample_mpes, we would just postpone the problem, no?

Reply:
Good point @rettigl indeed my suggestion to go towards NXsample_mpes was just a offer a helping hand to @domna to prevent having to merge this before his vacation THE way how this should be solved is BY PULLING the outdated and too simplistic NXsample apart based on all our combined knowledge in FAIRmat.